robot/basestation/templates/AsimovOperation.html

```diff
@@ -0,0 +1,413 @@
+                                    <th scope="col"> Motor Position
+                                        <br/>(Displacement)
+                                    </th>
+                                    <th scope="col">Motor Current (A)</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <tr>
+                                    <th scope="row">Stepper
+                                        <br/>Motor 1
+                                    </th>
+                                    <!-- add backend code to update status/motor position/motor current-->
+                                    <td>ALIVE</td>
+                                    <td>23</td>
+                                    <td>2.50</td>
+                                </tr>
+                                <tr>
+                                    <th scope="row">DC
+                                        <br/>Motor 2
+                                    </th>
+                                    <td>ALIVE</td>
+                                    <td>12</td>
+                                    <td>11.00</td>
+                                </tr>
+                                <tr>
+                                    <th scope="row">Stepper
+                                        <br/>Motor 3
+                                    </th>
+                                    <td>DEAD</td>
+                                    <td>84</td>
+                                    <td>7.45</td>
+                                </tr>
+                                <tr>
+                                    <th scope="row">Stepper
+                                        <br/>Motor 4
+                                    </th>
+                                    <td>DEAD</td>
+                                    <td>88</td>
+                                    <td></td>
+                                </tr>
+                                <tr>
+                                    <th scope="row">Arm Servo
+                                        <br/>Motor 1
+                                    </th>
+                                    <td>ALIVE</td>
+                                    <td>20</td>
+                                    <td>5.60</td>
+                                </tr>
+                                <tr>
+                                    <th>Arm Servo
+                                        <br/>Motor 2
+                                    </th>
+                                    <td>ALIVE</td>
+                                    <td>67</td>
+                                    <td>12.34</td>
+                                </tr>
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+            </div>
+        </div>
+        
+        <!-- Optional JavaScript -->
```

Comment:
Apparently, leaving `<script>` tags at the bottom of html files is "antiquated" and the "modern approach" is to use the `async` keyword, so that the scripts can be executed asynchronously, therefore ASAP but not blocking the rest of the page from loading. In case you're curious for the full scoop, check this [question/answer](https://stackoverflow.com/questions/436411/where-should-i-put-script-tags-in-html-markup) on stackoverflow.